File: TestTabLayout/app/src/main/java/com/pix/testtablayout/MainActivity.java
package com.pix.testtablayout;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

import com.pix.testtablayout.test.TestTabLayout01Activity;
import com.pix.testtablayout.test.TestTabLayout02Activity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void test01Click(View v) {
        startActivity(new Intent(this,TestTabLayout01Activity.class));
    }

    public void test02Click(View v) {
        startActivity(new Intent(this, TestTabLayout02Activity.class));
    }
}

File: TestTabLayout/app/src/main/java/com/pix/testtablayout/MainActivity.kt
package com.pix.testtablayout

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View

import com.pix.testtablayout.test.TestTabLayout01Activity
import com.pix.testtablayout.test.TestTabLayout02Activity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun test01Click(v: View) {
        startActivity(Intent(this, TestTabLayout01Activity::class.java))
    }

    fun test02Click(v: View) {
        startActivity(Intent(this, TestTabLayout02Activity::class.java))
    }
}
